install/README.md

```diff
@@ -84,6 +84,7 @@ Manual installation instructions are available for [these distributions](#suppor
 
 ## Installing on a Cloud Service Platform
 * [Amazon Web Services (AWS)](aws-installation-guide.md)
+* [Azure](azure-installation-guide.md)
```

Comment:
Oh wait - shouldn't we say "Microsoft* Azure"?

Reply:
I'm not sure -- this came from Sarah initially - I'll leave it to her discretion.  @scooley - WDYT?

Reply:
All the other providers are listed, so I think for consistency alone we should specify the providing company name.

Reply:
I vote we should add 'Microsoft' in the phrase - that is how it is called on their site, and they no doubt have some trademark or copyrights, so we should ack here.